server/src/main/java/com/flipkart/varadhi/db/ZKMetaStore.java

```diff
@@ -44,8 +45,8 @@ void createZNode(ZNode znode) {
     <T extends MetaStoreEntity> void createZNodeWithData(ZNode znode, T dataObject) {
         try {
             String jsonData = JsonMapper.jsonSerialize(dataObject);
-            String response =
-                    zkCurator.create().withMode(CreateMode.PERSISTENT).forPath(znode.getPath(), jsonData.getBytes());
+            String response = zkCurator.create().withMode(CreateMode.PERSISTENT)
+                    .forPath(znode.getPath(), jsonData.getBytes(StandardCharsets.UTF_8));
```

Comment:
charset needed ? if yes, should it be provided in JsonMapper utility ?

Reply:
I think yes. 
I have created another backlog for this fix:
https://github.com/flipkart-incubator/varadhi/issues/91.

So we can fix it (relating to jsonMapping) all there.

There is one more backlog around supporting schema change in MetaStoreEntity. This looks important.
https://github.com/flipkart-incubator/varadhi/issues/93